### ISSUE TITLE
Music skips songs without command almost always

### ISSUE THREAD

**Issue by DrEngi** [DrEngi/lmao-bot#81]
?!?!?fdasfasdfasdf

```
[tls @ 0x55d3ed6d47a0] Error in the pull function.
[matroska,webm @ 0x55d3ed6d1ee0] Read error
[tls @ 0x55d3ed6d47a0] The specified session has been invalidated for some reason.
    Last message repeated 1 times
2019-01-16 00:42:51,461 INFO: Preparing to terminate ffmpeg process 21474.
2019-01-16 00:42:51,462 INFO: ffmpeg process 21474 successfully terminated with return code of 0.
```

**Comment by DrEngi** [comment]
Fixed by 1.3.2

**Comment by DrEngi** [comment]
lol I lied it still isn't fixed
